dev-nais.yaml

```diff
@@ -27,3 +27,4 @@ spec:
   prometheus:
     enabled: true
     path: /kontakt-oss-api/internal/actuator/prometheus
+  webproxy: true
```

Comment:
Husker ikke hva `webproxy` brukes til. Hvorfor måtte den settes til `true` her?

Reply:
Den brukes for å kalle endepunkter på Internett. Salesforce ligger ikke i FSS